jnosql-lite/mapping-lite-column-test/src/test/java/org/eclipse/jnosql/lite/mapping/entities/PersonRepositoryTest.java

```diff
@@ -196,8 +206,8 @@ void shouldReturnFalseIfEntityDoesNotExistById() {
 
 
     @Test
-    void shouldFindAllEntitiesWithPageable() {
-        Pageable pageable = mock(Pageable.class);
+    void shouldFindAllEntitiesWithPageRequest() {
+        PageRequest pageable = mock(PageRequest.class);
```

Comment:
Could you also rename the variable name?

```suggestion
        PageRequest pageRequest = mock(PageRequest.class);
```